app/forms/keyword_form.rb

```diff
@@ -4,4 +4,7 @@
 class KeywordForm < ApplicationForm
   attribute :text, :string
   validates :text, presence: true, if: :deposit?
+  attribute :uri, :string
+  validates :uri, url: true, allow_blank: true
```

Comment:
Verify the URI is a valid URI using a custom validator defined below.

Reply:
Is there a path by which we would not get a valid URI? What would happen if this failed?

(My general philosophy has been to use validations for validating form input by the user only.)

Reply:
OCLC data error, I suppose? If it failed, I don't think the user would have any recourse. I'll yank the validator, because I agree w/ your philosophy.